gradle-mvn-push.gradle

```diff
@@ -103,7 +103,7 @@ afterEvaluate { project ->
             source = variant.javaCompile.source
             ext.androidJar = project.files(android.getBootClasspath().join(File.pathSeparator))
             classpath = files(variant.javaCompile.classpath.files) + files(ext.androidJar)
-            exclude '**/BuildConfig.java'
+            exclude '**/BuildConfig'
```

Comment:
For some reason those exclusions don't work with `.java` and gradle gives out warnings.